OWML.ModLoader/ModFinder.cs

```diff
@@ -48,20 +48,6 @@ private IModData GetModData(IModManifest manifest)
                     || (config == null && defaultConfig != null && defaultConfig.RequireVR);
                 storage.Save(manifest, Constants.ModManifestFileName);
             }
-            if (config == null && defaultConfig == null)
-            {
-                config = new ModConfig();
-                defaultConfig = new ModConfig();
-            }
-            else if (defaultConfig != null && config == null)
-            {
-                config = defaultConfig;
-            }
```

Comment:
removing de-nullifying probably wasn't neccessary, returning it would make `App.IsEnabled` unnecessary, but would make `FixConfigs` make unnecessary `MakeConsistentWithDefaults` calls

Reply:
(also I probably should have named `App.IsEnabled` `IsModEnabled` instead

Reply:
i don't get it 🤷‍♂️

Reply:
I guess it's not really a thing anymore because we added ModData.Enabled

Reply:
ok, so, resolve this? 😋